compose.yml

```diff
@@ -1,4 +1,13 @@
 version: '3.7'
+
+volumes:
+  pg-backend:
+  pg-sources:
+  pg-notifications:
+  redis:
+  zookeeper:
+  kafka:
+
```

Comment:
Volumes will be automatically created from now on. Use `docker volume` subcommands to inspect, list and delete them.